eng/mgmt/automation/generate.py

```diff
@@ -148,6 +148,8 @@ def update_changelog(changelog_file, changelog):
 
 def compare_with_maven_package(sdk_root, service, stable_version,
                                current_version):
+    logging.info('[Changelog] Compare stable version {0} with current version {1}'.format(stable_version, current_version))
```

Comment:
Log is like: 
2021-09-16 13:20:38 INFO [Changelog] Compare stable version 1.0.0 with current version 1.1.0-beta.2